custom/inddex/reports/gaps_detail.py

```diff
@@ -1,3 +1,5 @@
+import textwrap
```

Comment:
:exploding_head: I did not know about this. I've been using `inspect.cleandoc()` to unindent things. This library is cool.

Reply:
Oh yeah, I love it!